.rubocop.yml

```diff
@@ -1,546 +1,133 @@
-require: rubocop-rspec
```

Comment:
Please do a rebase against the master branch, because this file is managed by ```voxpupuli/modulesync```, ```voxpupuli/modulesync_config```.